src/Controller/controllers.go

```diff
@@ -129,7 +128,7 @@ func (c Controller) LoadFile(w http.ResponseWriter,
 		}
 	}
 	if errorFlag == true {
```

Comment:
no need for errorFlag or msg can change dirrectly error field of pad at err check (if err==nil) and return there